composer.json

```diff
@@ -20,9 +20,10 @@
     },
     "minimum-stability": "dev",
     "require": {
-        "illuminate/support": "^5.1"
+        "php": "^5.6|^7.0",
```

Comment:
Could you please replace the single pipes with doubles?
Based on the [documentation](https://getcomposer.org/doc/articles/versions.md#version-range), that's the recommended "or" operator